src/app/Core/api.service.ts

```diff
@@ -0,0 +1,20 @@
+export interface ApiResponse<T>{
+    apiStatus: number;
+    apiProp?: T;
+}
+
+export async function ApiService(apiUrl: string): Promise<ApiResponse<Record<string, unknown>[]>> {

```

Comment:
Consider naming your service as functions because your services are functions, e.g.:
`callApiService()` or `fetchJsonFromApi()`
Nouns like `ApiService` indicate a class like usage: `ApiService.fetch()`
So i would suggest for later tickets to use classes for Services, then you can also encapsulate multiple functions in one service class:
```
class ApiService {
  public static fetchFromUrl(url: string) {...}
  public static fetchFromFile(fileName: string) {...}
}
const foo = ApiService.fetchFromUrl('fooUrl');
```

Reply:
Renameing done.